react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -235,20 +250,21 @@ export default class JabraService extends VendorImplementation {
   async outgoingCall (callInfo: CallInfo): Promise<void> {
     try {
       this.callLock = await this.callControl.takeCallLock();
-      if (this.callLock) {
-        this.pendingConversationId = callInfo.conversationId;
-        this.pendingConversationIsOutbound = true;
-        this.callControl.offHook(true);
-        return Promise.resolve();
-      }
     } catch ({ message, type }) {
       if (this.checkForCallLockError(message, type)) {
         this.logger.info(message);
-        this.callControl.offHook(true);
+        this.callLock = true;
       } else {
         this.logger.error(type, message);
       }
     }
+
+    if (this.callLock) {
+      this.pendingConversationId = callInfo.conversationId;
+      this.pendingConversationIsOutbound = true;
+      this.callControl.offHook(true);
+      return Promise.resolve();
```

Comment:
This return is pointless

Reply:
Totally right, that was a relic that I overlooked to remove